smaug/python/ops/activation_ops.py

```diff
@@ -3,41 +3,73 @@
 from smaug.python import global_vars
 from smaug.python.ops import common
 
-def set_activation_params(activation, act_params_proto, act_params):
-  if activation not in global_vars.supported_activations:
-    raise AssertionError(
-        ("%s is not a supported activation function. "
-         "Supported activations: %s.")
-        % (OpType.Name(activation),
-           [OpType.Name(a) for a in supported_activations]))
-  if act_params != None:
+def _get_activation_type(activation):
+  """Return the corresponding activation type.
+
+  Args:
+    activation: A string.
+
+  Returns:
+    An activation op type such as `types_pb2.ReLU`.
+  """
+  if activation == "relu":
```

Comment:
Convert this to a dict keyed by the enum instead.

Reply:
Done.